src/main/resources/api/service_openapi.yaml

```diff
@@ -739,6 +756,162 @@ components:
         memberEmail:
           type: string
 
+    CreateControlledGoogleBucketRequestBody:
+      description: Payload for requesting a new controlled resource.
+      type: object
+      properties:
+        jobId:
+          description: Asyncronous job ID for this create operation.
+          type: string
+        googleBucket:
+          $ref: '#/components/schemas/GoogleBucketCreationParameters'
+
+    CreatedControlledGoogleBucket:
+      description: Response Payload for requesting a new controlled resource.
+      type: object
+      properties:
+        resourceId:
+          description: UUID of a newly-created resource. Null if not created yet.
+          type: string
+          format: uuid
+        jobId:
+          description: Asyncronous job ID for this create operation.
+          type: string
+        googleBucket:
+          $ref: '#/components/schemas/GoogleBucketCreationParameters' # TODO: should this include all parameters, not just the settable ones we expose?
+
+    GoogleBucketCreationParameters:
+      description: >-
+        Bucket-specific properties to be set on creation. These are a subset of the
+        values accepted by the Google Storage API.
+      type: object
+      properties:
+        name:
+          description: A valid bucket name per https://cloud.google.com/storage/docs/naming-buckets.
+          type: string
+        location:
+          description: A valid bucket location per https://cloud.google.com/storage/docs/locations.
+          type: string
+        defaultStorageClass:
+          $ref: '#/components/schemas/GoogleBucketDefaultStorageClass'
+        lifecycle:
+          $ref: '#/components/schemas/GoogleBucketLifecycle'
+
+    GoogleBucketDefaultStorageClass:
+      description: >-
+        Type and availability of objects in a bucket, described at https://cloud.google.com/storage/docs/storage-classes.
+        This list does not include legacy types that apply to some existing buckets. For those, see
+        GoogleBucketLegacyStorageClass.
+      type: string
+      enum:
+        - STANDARD
+        - NEARLINE
+        - COLDLINE
+        - ARCHIVE
+    GoogleBucketLegacyStorageClass:
```

Comment:
We won't need to support this, since we only make new bucket.

Reply:
Oh.